instrumentation/jetty/jetty-11.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jetty/v11_0/Jetty11Singletons.java

```diff
@@ -26,10 +23,7 @@
       INSTRUMENTER =
           ServletInstrumenterBuilder.<HttpServletRequest, HttpServletResponse>create()
               .addContextCustomizer(
-                  (context, request, attributes) -> {
-                    context = ServerSpanNaming.init(context, CONTAINER);
-                    return new AppServerBridge.Builder().init(context);
-                  })
+                  (context, request, attributes) -> new AppServerBridge.Builder().init(context))
```

Comment:
should `ServerSpanNaming.get()` context customizer be added here? I notice it's missing in some cases and included in others, but not clear to me why

Reply:
It actually is added, via the `ServletInstrumenterBuilder` class.
All server instrumentations should add `ServerSpanNaming.get()`